src/DockManager.h

```diff
@@ -145,7 +145,7 @@ class ADS_EXPORT CDockManager : public CDockContainerWidget
 	{
 		ActiveTabHasCloseButton = 0x0001,    //!< If this flag is set, the active tab in a tab area has a close button
 		DockAreaHasCloseButton = 0x0002,     //!< If the flag is set each dock area has a close button
-		DockAreaCloseButtonClosesTab = 0x0004,//!< If the flag is set, the dock area close button closes the active tab, if not set, it closes the complete cock area
```

Comment:
funny typo 😃

Reply:
Do you think the project owner will approve this change? :)